modules/renter/hostdb/consts.go

```diff
@@ -59,10 +59,10 @@ var (
 
 	// scanningThreads is the number of threads that will be probing hosts for
 	// their settings and checking for reliability.
-	scanningThreads = build.Select(build.Var{
-		Standard: int(20),
-		Dev:      int(4),
-		Testing:  int(3),
+	maxScanningThreads = build.Select(build.Var{
+		Standard: int(40),
+		Dev:      int(8),
+		Testing:  int(6),
```

Comment:
Dev and testing should be kept to 3 and 4 - those numbers are because it's tough to spin up enough hosts to exceed them. Standard is kept to 40 to limit system load (your new code makes it acceptable to raise to 40)